backend/src/ElectronApp.ts

```diff
@@ -125,7 +125,6 @@ export default class ElectronApp {
   }
 
   private openWindow = (openDevTools?: boolean) => {
-    console.log('open window')
```

Comment:
Do you really want to keep this console.log

Reply:
No - it's been removed in another pr: https://github.com/remoteit/desktop/compare/DESK-315